cmd/conformance-tests/main.go

```diff
@@ -164,58 +169,66 @@ const (
 var defaultTimeout = 10 * time.Minute
 
 var (
-	providers  string
-	pubKeyPath string
-	sversions  string
+	providers             string
+	pubKeyPath            string
+	sversions             string
+	sdistributions        string
```

Comment:
The s prefix is probably because these are the stringified versions of the CLI flags, that we have so that the Options struct has properly parsed, usable configuration fields.

> and if it has a meaning can you please add it in the usage section in the args messages.

What do you mean by that? Is

```go
flag.StringVar(&sdistributions, "distributions", "", "a comma-separated list of distributions to test (cannot be used in conjunction with -exclude-distributions)")
```

not enough?

Reply:
> The s prefix is probably because these are the stringified versions of the CLI flags

No it is fine.